meinberlin/apps/votes/forms.py

```diff
@@ -9,11 +9,14 @@ class VotingTokenWidget(forms.MultiWidget):
     def __init__(self, attrs=None):
         widgets = (
             forms.TextInput(attrs={'minlength': '4',
-                                   'maxlength': '4'}),
+                                   'maxlength': '4',
```

Comment:
this can be removed as it is being refactors as I looked into it and having multiple inputs is bad a11y so we will make a single input